plugins/modules/route53.py

```diff
@@ -495,6 +547,7 @@ def main():
         identifier=dict(type='str'),
         weight=dict(type='int'),
         region=dict(type='str'),
+        geo_location=dict(type='dict', required=False),
```

Comment:
Have you considered adding the valid suboptions to the argument spec? That would prevent typos/incorrect subtypes.

Reply:
Done.